### PR TITLE
Notification styles figma

### PR DESCRIPTION
**Pull request by ShashankAnantharam**
Added relevant notification box styles

### CODE REVIEW

File: src/styles/EmailForm.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.emailform-notification-title{
```
Comment (JSAssassin) [inline src/styles/EmailForm.css]: can this go inside the one below, that has the same class name?

**Review by JSAssassin — CHANGES_REQUESTED** [review]
The design on figma is rectangular. Can we change this to look like that. Ty.

File: src/styles/EmailForm.css (hunk 1 above)
Comment (ShashankAnantharam) [reply src/styles/EmailForm.css]: That  is  a different  class. I should be header instead of title. I fixed it

**Comment by JSAssassin** [comment]
Thank you!

**Review by JSAssassin — APPROVED** [review]

**Pull request merged**